src/main/groovy/com/f2prateek/javafmt/AndroidJavaFmtExtension.groovy

```diff
@@ -0,0 +1,5 @@
+package com.f2prateek.javafmt;
+
+class AndroidJavaFmtExtension {
+  boolean aospStyle = false
```

Comment:
boolean instead of def just comes from my java habits.
How do you want to handle unknown enum values when Google adds another style option? Fail or default to `Style.GOOGLE`?